vignettes/tidytranscriptomics.Rmd

```diff
@@ -176,11 +169,75 @@ counts <-
 counts
 ```
 
-## Filtering lowly transcribed genes  
-
-Genes with very low counts across all libraries provide little evidence for differential expression and they can interfere with some of the statistical approximations that are used later in the pipeline. They also add to the multiple testing burden when estimating false discovery rates, reducing power to detect differentially expressed genes. These genes should be filtered out prior to further analysis.
-
-We can perform the filtering using tidybulk `keep_abundant` or `identify_abundant`. These functions can use the *edgeR* `filterByExpr` function described in [@law2016rna] to automatically identify the genes with adequate abundance for differential expression testing. By default, this will keep genes with ~10 counts in a minimum number of samples, the number of the samples in the smallest group. In this dataset the smallest group size is four (as we have four dex-treated samples versus four untreated). Alternatively, we could use `identify_abundant` to identify which genes are abundant or not (TRUE/FALSE), rather than just keeping the abundant ones. 
+## tidybulk
```

Comment:
I wonder if we should delete this table of tidybulk functions? As just had a look at the workshop description that was submitted and the emphasis is on tidySummarisedExperiment and tidySingleCellExperiment more so than tidybulk? Or include tables of functions for those packages? See below

Recently, plyranges and tidybulk represented efforts toward the harmonization of transcriptomic data structures and workflows using the concept of data tidiness, to facilitate modularisation. In this workshop, we present tidySingleCellExperiment and tidySummarizedExperiment, two R packages that allow the user to visualise and manipulate SingleCellExperiment and SummarizedExperiment objects in a tidy fashion. Importantly, the framework tidybulk now works natively with SummarizedExperiment objects and, thanks to tidySummarizedExperiment, allow tidy and modular RNA sequencing analyses without renouncing to the efficiency of Bioconductor data containers. These tools are part of the tidytranscriptomic R software suite, and represent an effort toward the harmonisation of transcriptional analyses under the tidy umbrella.

Reply:
Good point. To add a table of function for those two would be nice, so to give a glance at the capabilities and help the users to get oriented in the following material.

I'm good with dropping tables for tidybulk.

Reply:
Ok cool, we can add the tables later as at the moment we just need one Docker build contained the workshop skeleton by Friday. I'm traveling Fri and Sat so can only work on this today but hopefully we can get the build here to work by then.

Reply:
Amazing.